flocker/node/agents/pd.py

```diff
@@ -0,0 +1,383 @@
+# -*- test-case-name: flocker.node.agents.functional.test_pd -*-
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+A PD implementation of the ``IBlockDeviceAPI``.
```

Comment:
Sounds good.  We've worked to convince ourselves that we'll use "the most recognizable and reasonable product name" as the module name.